pkg/config/schema.go

```diff
@@ -0,0 +1,105 @@
+package config
+
+import "github.com/xeipuuv/gojsonschema"
+
+var decodeClientPayloadSchema *gojsonschema.Schema
+
+const requestCapabilities = `
```

Comment:
Usually consts are declared after vars in the rest of codebase.

Reply:
@xla I think you mean before. but you're right

Reply:
Ooops, yes correctly.